src/components/shared/UserInfoPopover/UserInfoPopover.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+// import isEmpty from 'lodash/isEmpty';
+// import { BigNumber } from 'ethers';
+
+// import { SpinnerLoader } from '~shared/Preloaders';
+import Tag from '~shared/Tag';
+// import {
+//   useColonyNativeTokenQuery,
+//   useUserBalanceWithLockQuery,
+// } from '~data/index';
+// import { getAllUserRoles } from '~redux/transformers';
+import UserInfo from './UserInfo';
+import NotAvailableMessage from '../NotAvailableMessage/NotAvailableMessage';
+import { User, Colony } from '~types';
+import { useAppContext, useUserReputationForTopDomains } from '~hooks';
+
+// import UserPermissions from './UserPermissions';
+// import UserTokens from './UserTokens';
+import UserReputation from './UserReputation';
+
+import styles from './UserInfoPopover.css';
+
+interface Props {
+  colony?: Colony;
+  user?: User;
+  banned?: boolean;
+}
+
+const displayName = 'UserInfoPopover';
+
+const UserInfoPopover = ({
+  // colony: { colonyAddress },
+  colony,
+  user,
+  banned = true,
```

Comment:
Why is this `true` by default? This seems wrong.

Reply:
yes, it should be false, I must have set that to true while testing the Tag component. Nice catch.